src/panels/config/automation/action/types/ha-automation-action-wait_for_trigger.ts

```diff
@@ -52,6 +55,17 @@ export class HaWaitForTriggerAction
     `;
   }
 
+  private _timeoutChanged(ev: CustomEvent): void {
```

Comment:
That comment was supposed to be for this line on typing `ev`.  Commenting on lines in the mobile app is horrible with VoiceOver.

The diff is showing the whole method was added.  Is that not right?

Reply:
> Commenting on lines in the mobile app is horrible with VoiceOver.

Uh comments on larger diff sometimes look horrible from the UI as well (it looked like the comment was on the part below).

Anyways, yeah, understood and agree 👍 

I've pushed a change to make the typing more specific.